.travis.yml

```diff
@@ -32,10 +29,13 @@ matrix:
       env:
         - SYMFONY_VERSION='2.7.*@dev'
         - FRAMEWORK_EXTRA_VERSION='~3.0'
-    - php: 5.6
-      env:
-        - PHPUNIT_FLAGS="--coverage-clover=coverage.clover"
-        - COVERAGE=true
+
+php:
+  - 5.3
+  - 5.4
+  - 5.5
+  - 7.0
+  - hhvm
```

Comment:
changing this is useless (the order of top-level keys is meaningless). and keeping the list of PHP versions on top near the definition of the language makes more sense IMO